app/models/dfe_sign_in_user.rb

```diff
@@ -47,7 +47,15 @@ def self.load_from_session(session)
 
   def user
     # TODO: When dfe sign-in is fully implemented, we will be able to find the user by the id == id_token.
-    @user ||= user_klass.find_by(email:)
+    @user ||= begin
+      user = user_klass.find_by(email:)
+
+      if service == :placements && user&.support_user?
+        user = user.becomes(Placements::SupportUser)
+      end
```

Comment:
It feels _slightly weird_ to be doing this here. But I figured the `DfeSignInUser` model already has an awareness of the split between services and how that affects the type of User class returned. So this seems like the most obvious place to put it.